frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -25,3 +25,16 @@ export const OCSServiceModel: K8sKind = {
   id: 'ocscluster',
   crd: true,
 };
+
+export const VolumeSnapshotModel: K8sKind = {
+  label: 'Volume Snapshot',
+  apiVersion: 'snapshot.storage.k8s.io/v1alpha1',
+  apiGroup: 'snapshot.storage.k8s.io',
+  plural: 'volumeSnapshots',
```

Comment:
I'd expect this to be

```suggestion
  plural: 'volumesnapshots',
```